html/elements/input.json

```diff
@@ -384,7 +384,7 @@
         },
         "dirname": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/input#dirname",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Attributes/dirname",
```

Comment:
This may not be desired, but it kind of makes sense.

Reply:
Fixed in [036c74a](https://github.com/mdn/browser-compat-data/pull/25563/commits/036c74a1d18f9d6d352e80cc72572eb13e43513a).